src/components/LinksList.vue

```diff
@@ -275,10 +275,10 @@ export default {
                   this.items = this.grabDois(data);
                 }
                 // eslint-disable-next-line no-case-declarations
-                const uniqueCitations = typeof meta.uniqueCitations !== 'undefined'
-                  ? meta.uniqueCitations[0].citations
-                  : 0;
-                this.$emit('citationsLoaded', uniqueCitations);
+                // const uniqueCitations = typeof meta.uniqueCitations !== 'undefined'
```

Comment:
Maybe just remove the commented code if it's dead code, git history preserves what it was.